File: packages/pensopay_android/android/src/main/kotlin/com/flutter/pensopay/networking/pensopayapi/pensopaylink/models/PPShipping.kt
package com.flutter.pensopay.networking.pensopayapi.pensopaylink.models

import org.json.JSONObject

class PPShipping: JSONObject() {

    // Optional Properties

    var method: String? = null
    var company: String? = null
    var amount: Int? = null
    var vat_rate: Double? = null
    var tracking_number: String? = null
    var tracking_url: String? = null

}

File: packages/pensopay_ios/ios/Classes/Networking/Pensopay API/Pensopay Link/Models/PPShipping.swift
//
//  PPShipping.swift
//  PensopaySDK
//
//  Created on 07/26/2022
//  Copyright © 2022 Pensopay. All rights reserved.
//

import Foundation

public class PPShipping : Codable {

    // MARK: - Properties
    
    public var method: String?
    public var company: String?
    public var amount: Int?
    public var vat_rate: Double?
    public var tracking_number: String?
    public var tracking_url: String?
    
    
    // MARK: Init
    
    public init() {
        
    }
    
}
